src/features/presence/PresenceCollaboratorsList.js

```diff
@@ -197,7 +144,15 @@ class PresenceCollaboratorsList extends React.Component<Props> {
                     onScroll={this.throttledHandleScroll}
                     role="list"
                 >
-                    {collabList}
+                    {collaborators.map(collaborator => (
+                        <PresenceCollaborator
+                            collaborator={collaborator}
+                            disableHover
+                            intl={intl}
```

Comment:
Do we need to pass this down if we're using `injectIntl` in the PresenceCollaborator component?